helix-rest/src/main/java/org/apache/helix/rest/server/resources/zookeeper/ZooKeeperAccessor.java

```diff
@@ -62,14 +62,20 @@ public Response get(@PathParam("path") String path, @QueryParam("command") Strin
     if (cmd == null) {
       return badRequest("Invalid ZooKeeper command: " + commandStr);
     }
+    if (path == null) {
+      return badRequest("Path is missing in the request!");
+    }
 
     // Lazily initialize ZkBaseDataAccessor
     ServerContext _serverContext =
         (ServerContext) _application.getProperties().get(ContextPropertyKeys.SERVER_CONTEXT.name());
     _zkBaseDataAccessor = _serverContext.getByteArrayZkBaseDataAccessor();
 
-    // Need to prepend a "/" since JAX-RS regex removes it
-    path = "/" + path;
+    if (path.isEmpty() || path.charAt(0) != '/') {
```

Comment:
This path is getting from URL. Why we have different behavior? Sometimes it has slash sometimes not?